images/win/toolsets/toolset-2016.json

```diff
@@ -231,5 +231,14 @@
         ],
         "vsix": [
         ]
+    },
+    "docker": {
+        "images": [
+            "mcr.microsoft.com/windows/servercore:ltsc2016",
+            "mcr.microsoft.com/windows/nanoserver:10.0.14393.953",
+            "mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2016",
+            "mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2016",
+            "microsoft/aspnetcore-build:1.0-2.0"
```

Comment:
This image "microsoft/aspnetcore-build" is deprecated

Reply:
Unfortunately we cannot remove this image because it may cause regression